config/routes.rb

```diff
@@ -32,6 +32,7 @@ def matches?(request)
 
         # authn-oidc has to be first as it can be ambgiuous with the optional :service_id & :id
         post '/authn-oidc(/:service_id)/:account/authenticate' => 'authenticate#authenticate_oidc'
+        post '/authn-gce/:account/authenticate' => 'authenticate#authenticate_gce'
```

Comment:
we decided against a service_id here? even if it's optional?

Reply:
note that we also don't have an id so we need an explicit route for `authn-gce`